src/util/cmdlineargs.h

```diff
@@ -67,16 +68,16 @@ class CmdlineArgs {
 
   private:
     CmdlineArgs() :
-        m_startInFullscreen(false), //Initialize vars
+        m_startInFullscreen(false), // Initialize vars
         m_midiDebug(false),
         m_developer(false),
         m_safeMode(false),
-        m_settingsPath(QDir::homePath().append("/").append(SETTINGS_PATH)) {
+        m_settingsPath(QDesktopServices::storageLocation(QDesktopServices::DataLocation).append("/")) {
```

Comment:
ok, I reverted Linux and MacOSX changes and made this PR a windows-only one. I think we are not ready for a change on linux and MacOSX.